devices.js

```diff
@@ -26,6 +26,7 @@ const tz = require('./converters/toZigbee');
 const utils = require('./converters/utils');
 const globalStore = require('./converters/store');
 const ota = require('./ota');
+const exposes = require('./lib/exposes');
```

Comment:
Kindly export exposes from main package, so it could be reused in external converters